README.md

```diff
@@ -1,2 +1,10 @@
 # data-testing
 Light-weighted data for testing/tutorial
+
+# Content
+* sub-unf01: Bids folder with multiple contrast (T2w,T2star and T1w). Original image from [ivadomed spine generic example repo](https://github.com/ivadomed/data_example_spinegeneric/releases/tag/r20200907) resampled to 1mm isotropic and converted to float32 to reduce repo size. Subject was randomly selected among the available subjects.
```

Comment:
it says "sub-test001" in the bounding_box_dict.json file

Reply:
Changed in 9bce745